.github/workflows/test-run-enos-scenario-matrix.yml

```diff
@@ -49,26 +49,29 @@ jobs:
       sample: ${{ steps.metadata.outputs.sample }}
       vault-version: ${{ steps.metadata.outputs.vault-version }}
     steps:
-      - uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
         with:
           ref: ${{ inputs.vault-revision }}
       - uses: hashicorp/action-setup-enos@v1
         with:
           github-token: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
       - id: metadata
         run: |
-          echo "build-date=$(make ci-get-date)" >> "$GITHUB_OUTPUT"
-          sample="$(enos scenario sample observe ${{ inputs.sample-name }} --chdir ./enos --min 1 --max ${{ inputs.sample-max }} --seed "$(date +%s%N)" --format json | jq -c ".observation.elements")"
-          echo "sample=$sample"
-          echo "sample=$sample" >> "$GITHUB_OUTPUT"
-          if [[ "${{ inputs.vault-edition }}" == "oss" ]]; then
-            echo "vault-version=${{ inputs.vault-version }}" >> "$GITHUB_OUTPUT"
+          build_date=$(make ci-get-date)
```

Comment:
This is fine. I think we probably need to sync our Makefile with main to get all of the targets with the updated build workflows.